regtests/ww3_tp2.15/input/ww3_ounp_par.nml

```diff
@@ -78,6 +78,7 @@
 !                                          ! 4: Spectral partitioning
 !     SPECTRA%SCALE_FAC     = 1            ! Scale factor (-1=disabled)
 !     SPECTRA%OUTPUT_FAC    = 0            ! Output factor (0=normalized)
+!     SPECTRA%TYPE          = 4            ! [2 = SHORT, 3 = it depends , 4 = REAL]
```

Comment:
I don't think this change is relevant to this PR.
Also, I think the addition of SPECTRA%TYPE to ww3_ounp came after our PS45 branch, so it describes a feature that does not exist in out branch.

Reply:
Totally agree, now this line was deleted.